CHANGELOG.md

```diff
@@ -39,6 +39,10 @@
   [Timofey Solonin](https://github.com/biboran)
   [#2435](https://github.com/realm/SwiftLint/issues/2435)
 
+  * Swiftlint now fails when not using pinned version.
```

Comment:
Please match the indentation and formatting of other entries. Also it's "SwiftLint" not "Swiftlint". Also needs two trailing spaces as defined in CONTRIBUTING.md